book/src/pong_tutorial/pong_tutorial_02.md

```diff
@@ -294,85 +314,90 @@ world
+get the size of the texture it needs to do its magic.
 
-```rust,ignore
-fn initialise_camera(world: &mut World) {
-    world
-        .create_entity()
-        .with(Camera::standard_2d())
-        .build();
-}
-```
+> Behind the scene, the `with_sprite` method adds `Mesh` and `Material`
+> components to your entity. It is a utility function to spare you from
+> the rendering details, but Amethyst can expose all the precision of
+> the rendering process if you need it. Many utility functions like this
+> exist in Amethyst to make prototyping easier.
 
-The camera is another entity, with a `Camera` component. Amethyst's
-[`standard_2d` camera][2d] uses an orthographic projection, and defines a
-screenspace coordinate system of `-1.0,-1.0` in the bottom left and `1.0,1.0` in
-the top right.
+> Here, we are using the `with_sprite` utility twice for the same sprite.
+> Keep in mind however that another syntax exists in the `SpriteRenderData` struct
+> when we need multiple entities to display the exact same sprite, leading
+> to improved performance. Here, however, it is negligible.
 
-If you want to to define your own coordinate system, you can use something like
-this:
+Now let's add our initialise functions to the `on_start` function in `impl State
+for Pong`. It now looks like this:
 
 ```rust,ignore
-fn initialise_camera(world: &mut World) {
-    world.create_entity()
-        .with(Camera::from(Projection::orthographic(0.0, WIDTH, HEIGHT, 0.0)))
-        .with(GlobalTransform(Matrix4::from_translation
-                (Vector3::new(0.0, 0.0, 1.0)).into())
-             )
-        .build();
+fn on_start(&mut self, data: StateData<GameData>) {
+    let world = data.world;
+
+    // Load the spritesheet necessary to render the graphics.
+    let spritesheet = {
+        let loader = world.read_resource::<Loader>();
+        let texture_storage = world.read_resource::<AssetStorage<Texture>>();
+        loader.load(
+            "img/pong_spritesheet.png",
+            PngFormat,
+            Default::default(),
+            (),
+            &texture_storage,
+        )
+    };
+
+    initialise_paddles(world, spritesheet);
+    initialise_camera(world);
 }
 ```
 
-To use that custom camera you'll need to define WIDTH and HEIGHT constants, and
-redo the position math in the `initialise_paddles` function.
+Okay! We've defined our `Paddle` component, and created two entities which have
+a `Paddle` component, a `GlobalTransform` component and a sprite. When our game
+starts, we'll add the left and right paddles to the world, along with a camera.
 
-Now let's add our initialise functions to the `on_start` function in `impl State
-for Pong`.
+Before we continue, one last note.
+Components do not have to be registered manually to be used, however you need to have
+something that uses them to have them be registered automatically.
+As nothing uses our `Paddle` component yet, we will register it manually before we intialise
```

Comment:
"initialize"

Reply:
There is another instance of "initialise" that I did not write tho lol
I'll fix it too.

Reply:
I think that's american vs british here right?

Reply:
Ah I'm stupid nvm

Reply:
The real reason this caught my attention was the missing second "i" so remember to get that too.